marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/entities/search/EntitySearchManagerTest.java

```diff
@@ -122,41 +122,6 @@ public void testSearchResultsOnNoData() {
         assertTrue(new EntitySearchManager(getHubClient()).search(query).get().isEmpty());
     }
```

Comment:
I believe we need two kinds of tests in this class:

1. One or more tests that verify that the correct "criteria" is constructed based on the SortOrder objects in the SearchQuery
1. A test that verifies that inserts two customer entity instances and verifies that sorting on a property in ascending and then descending order returns the 2 instances in the correct order.